orb.yml

```diff
@@ -128,3 +134,45 @@ jobs:
         format: << parameters.format >>
         attributes: << parameters.attributes >>
         repository: << parameters.repository >>
+
+examples:
+  publish_java:
+    description: "Example builds and publishes java artifact"
+    usage:
+      version: 2.1
+      jobs: # a collection of steps
+        build: # runs not using Workflows must have a `build` job as entry point
+
+          docker: # run the steps with Docker
+          - image: circleci/openjdk:8-jdk-browsers # ...with this image as the primary container; this is where all `steps` will run
+
+          steps: # a collection of executable commands
+
+          - checkout # check out source code to working directory
```

Comment:
It looks like these only need the `:` when there are sub elements. Is this correct?

Reply:
the magic/terror of yaml